components/api/flow.js

```diff
@@ -35,6 +39,10 @@ const Flow = ({ url }) => (
     <SectionLayout sub title="Installing the definitions">
       <CodeBlock code={install} />
     </SectionLayout>
+  
+    <p>If you are on npm >= 5.2 you could simply use npx</p>
```

Comment:
Sounds great. But I just looked through npx repo and it doesn't shoot the message quick enough to anyone that has never used `npx`. Someone aware of npx would get it but we may not be saving people without a pre-existing knowledge of what npx is from globally installing flow-typed. Just a thought. If you think having a link is the best way to go I could make that as well